docs_user/assemblies/openstack_adoption.adoc

```diff
@@ -1,5 +1,12 @@
+ifdef::context[:parent-context: {context}]
+
+[id="openstack-adoption_{context}"]
+
 = OpenStack adoption
-:toc: left
+
+:context: openstack-adoption
+//kgilliga: I don't think we need the toc and toclevels attributes.
```

Comment:
Will we still get a sidebar for navigation though? I find the sidebar useful.

Reply:
I guess we will because it's in main.adoc: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/main.adoc?plain=1#L2

Reply:
I was comparing this file with another "master.adoc"" (aka main.adoc) file, and I didn't see the toc attributes. I'm happy to leave them as is if they're necessary.